src/lib/openssl_crypto.c

```diff
@@ -95,6 +95,12 @@
 
 #include <botan/ffi.h>
 
+#define FAIL(str)  \
```

Comment:
It's multi (2) statement macro. Please find more here:
https://en.wikibooks.org/wiki/More_C%2B%2B_Idioms/Multi-statement_Macro